content/en/logs/log_collection/nodejs.md

```diff
@@ -128,6 +128,53 @@ logs:
     sourcecategory: sourcecode
 ```
 
+## Aggentless Logging
+
+It is possible to stream logs from your application to Datadog or to the Datadog Agent directly. This is not the recommended setup as handling connection issues should not be done directly in your application, but it might not be possible to log to a file when your application is running on a machine that cannot be accessed.
+
+{{< tabs >}}
+{{% tab "Winston 3.0" %}}
+
+You can use [winston HTTP transport][1] to send your logs directly through the datadog API.
```

Comment:
If we recommend this transport, should we take ownership of it?

Reply:
what do you mean by "take the ownership of it" ?

Reply:
I meant the other transport and not this one, sorry for the confusion, I'll copy this comment to the other one.